sensor/common/detector/enricher.go

```diff
@@ -143,6 +143,15 @@ outer:
 }
 
 func (c *cacheValue) scanAndSet(ctx context.Context, svc v1.ImageServiceClient, req *scanImageRequest) {
+	if features.UnqualifiedSearchRegistries.Enabled() {
+		// Guarding this with a feature flag because it's planned to be introduced in a
```

Comment:
Nit: A JIRA to track this would be ideal.

Reply:
https://github.com/stackrox/stackrox/pull/11359/commits/3c26bd345a045d153a6653d0eb294ece21ddfd13